Framework/PythonInterface/test/python/mantid/api/SpectrumInfoTest.py

```diff
@@ -2,24 +2,133 @@
 
 import unittest
 from testhelpers import WorkspaceCreationHelper
+from mantid.kernel import V3D
+from mantid.api import SpectrumDefinition
 
 class SpectrumInfoTest(unittest.TestCase):
 
     _ws = None
```

Comment:
interesting. Why did you create the workspace object at the class level?

Reply:
`SpectrumInfoTest.py` was actually partly implemented and this was there from before...
I just continued using it as I thought it was the right way?

Reply:
I think it's fine. I just wanted to make sure you had thought about it.